facebook/OpenBIC
# fby3.5: common: Fix ADM1278 VIN reading

Summary:
- Fix ADM1278 VIN reading loses fraction value problem.

Test plan:
- Build code: Pass
- Get ADM1278 VIN: Pass

Log:
1. Check class1 ADM1278 sensor reading
- Before fix
root@bmc-oob:~# sensor-util slot4 | grep HSC | grep V
HSC Input Vol                (0x29) :   12.00 Volts | (ok)

- After fix
root@bmc-oob:~# sensor-util slot4 | grep HSC | grep V
HSC Input Vol                (0x29) :   12.42 Volts | (ok)

2. Check class2 ADM1278 sensor reading
- Before fix
root@bmc-oob:~# sensor-util slot1 | grep HSC | grep V
HSC Input Vol                (0x29) :   12.00 Volts | (ok)
BB_HSC_VIN                   (0xDC) :   12.00 Volts | (ok)

- After fix
root@bmc-oob:~# sensor-util slot1 | grep HSC | grep V
HSC Input Vol                (0x29) :   12.15 Volts | (ok)
BB_HSC_VIN                   (0xDC) :   12.16 Volts | (ok)
